python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -555,6 +579,43 @@ def _fsproxy_slow(self) -> Any:
         self._fsproxy_wrapped = self._fsproxy_fast_to_slow()
         return self._fsproxy_wrapped
 
+    def set_transfer_block_state(
+        self, transfer_block: _BlockState | None = None
+    ):
+        """Set transfer blocking flag for this specific instance.
+
+        Parameters
+        ----------
+        transfer_block : _BlockState | None
+            None: No blocking (default)
+            _BlockState.TO_FAST: Block fast-to-slow transfers only
+            _BlockState.TO_SLOW: Block slow-to-fast transfers only
```

Comment:
This naming seems backwards. TO_FAST seems like it should block transfers from slow "to fast" as per the name, no?